assets/_scss/components/_footer.scss

```diff
@@ -0,0 +1,298 @@
+.usa-footer {
+
+  button {
+    width: 100%;
+  
+    @media (min-width: $tablet-up) {
+      width: auto;
+    }
+  }
+
+  .usa-unstyled-list {
+    display: block;
+  }
+
+  .usa-footer-primary-link {
+    border-top: 1px solid $color-base-link;  
+    color: $color-base-link;
+    display: block;
+    font-weight: $font-bold;
+    margin-top: 0;
+    padding: {
+      bottom: 2rem;
+      top: 2rem;
+    }
+    text-transform: uppercase;
+
+    @media (min-width: $tablet-up) {
+      border-top: none;
+      font-size: 1.9rem;
+      text-transform: none;
+    }  
+  }  
+}
+
+.usa-footer a {
+  font-weight: normal;
+}
+
+.usa-footer-return-to-top {
+  padding: {
+    bottom: 2rem;    
+    top: 2rem;
+  }
+}
+
+.usa-footer-primary-section {
+  background-color: $color-grey-100;
+
+  a, 
+  p, 
```

Comment:
Oh let me explain... If you remember we discussed adding left/right padding to our `grid-box` class so that content wouldn't run to the edge of the page. However, in this case, I needed content to run to the edge of the page bc the design dictates a horizontal line from end to end. So to accomplish this, I removed the padding from grid-box but then re-added it for the child elements.

There is probably a better way but this is what I came up with now. Let me know if you have another solution.